src/data-publication/reports/SupportingDocs.jsx

```diff
@@ -2,156 +2,204 @@ import React from 'react'
+                    'https://github.com/cfpb/hmda-platform/blob/master/docs/spec/markdown/modified_lar/2018_Modified_LAR_Data_Dictionary.md'
+                  }
+                  text='2018'
+                  title='Modified LAR 2018 data dictionary'
+                />
+              </li>
+              <li>
+                <ExternalLink
+                  url={
+                    'https://github.com/cfpb/hmda-platform/blob/master/docs/v1/2017_Modified_LAR_Data_Dictionary.csv'
+                  }
+                  text='2017'
+                  title='Modified LAR 2017 data dictionary'
+                />
+              </li>
+            </ul>
           </div>
+        </div>
+        <div>
+          <h4>Field Information</h4>
           <div>
-           <h4>Field Information</h4>
-            <div>
-              A discussion of the data available in the Modified LAR text files. Includes information on additional fields and how the base data was modified to protect the privacy of borrowers
-              <ul>
-                <li>
-                <a
-                  title="Modified LAR Field information"
-                  href="https://github.com/cfpb/hmda-platform/blob/master/docs/UsingModifiedLar.md"
-                >Field Information</a>
-                </li>
-              </ul>
-            </div>
+            A discussion of the data available in the Modified LAR text files.
+            Includes information on additional fields and how the base data was
+            modified to protect the privacy of borrowers
+            <ul>
+              <li>
+                <ExternalLink
+                  url={
+                    'https://github.com/cfpb/hmda-platform/blob/master/docs/UsingModifiedLar.md'
+                  }
+                  text='Field Information'
+                  title='Modified LAR Field information'
+                />
+              </li>
+            </ul>
           </div>
+        </div>
+        <div>
+          <h4>Code Sheet</h4>
           <div>
-            <h4>Code Sheet</h4>
-            <div>
-              A quick reference guide containing the valid codes by field, intended for use during analysis
-              <ul>
-                <li>
-                    <a
-                      title="Modified LAR Code Sheet"
-                      href="https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-public-LAR-code-sheet.pdf"
-                    >Code Sheet</a>
-                </li>
-              </ul>
-            </div>
+            A quick reference guide containing the valid codes by field,
+            intended for use during analysis
+            <ul>
+              <li>
+                <S3DocLink
+                  url={
+                    'https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-public-LAR-code-sheet.pdf'
+                  }
+                  title='Modified LAR Code Sheet'
+                  label='Code Sheet'
+                />
+              </li>
```

Comment:
1) S3DocLink has an `<li>` in it's implementation, so I'm seeing a browser error 
    <img width="531" alt="Screen Shot 2022-11-14 at 4 12 06 PM" src="https://user-images.githubusercontent.com/2592907/201787489-00da351c-abfe-435f-b09a-2a0a6db56017.png">

2) There's a lot of extra spacing to the left of the `Size` and `Updated` labels.  There is a rule in `SupportingDocs.css` that applies to all `div`s.  We probably want to make it more targeted.
    <img width="772" alt="Screen Shot 2022-11-14 at 4 10 59 PM" src="https://user-images.githubusercontent.com/2592907/201788459-885e6267-18f1-4854-9d51-f1de0a810535.png">